dspace-api/src/main/java/org/dspace/discovery/IndexClient.java

```diff
@@ -150,10 +168,55 @@ public static void main(String[] args) throws SQLException, IOException, SearchS
             indexer.optimize();
         } else if (line.hasOption('s')) {
             checkRebuildSpellCheck(line, indexer);
+        } else if (line.hasOption("item_uuid")) {
+            String itemUUID = line.getOptionValue("item_uuid");
+            Item item = ContentServiceFactory.getInstance().getItemService().find(context, UUID.fromString(itemUUID));
+            indexer.indexContent(context, item, line.hasOption("f"));
+        } else if (line.hasOption("u")) {
+            String optionValue = line.getOptionValue("u");
+            String[] identifiers = optionValue.split("\\s*,\\s*");
+            for (String id : identifiers) {
+                if (id.startsWith(ConfigurationManager.getProperty("handle.prefix")) || id.startsWith("123456789/")) {
+                    BrowsableObject dso = (BrowsableObject) HandleServiceFactory.getInstance()
+                            .getHandleService().resolveToObject(context, id);
+                    indexer.indexContent(context, dso, line.hasOption("f"));
+                }
+            }
+        } else if (line.hasOption('e')) {
+            try {
+                String filename = line.getOptionValue('e');
+                FileInputStream fstream = new FileInputStream(filename);
+                // Get the object of DataInputStream
+                DataInputStream in = new DataInputStream(fstream);
+                BufferedReader br = new BufferedReader(new InputStreamReader(in));
+                String strLine;
+                // Read File Line By Line
+
+                UUID item_id = null;
+                List<UUID> ids = new ArrayList<UUID>();
+
+                while ((strLine = br.readLine()) != null) {
+                    item_id = UUID.fromString(strLine.trim());
+                    ids.add(item_id);
+                }
+
+                in.close();
+
+                int type = -1;
+                if (line.hasOption('t')) {
+                    type = Integer.parseInt(line.getOptionValue("t"));
```

Comment:
This option isn't documented above.

Reply:
option removed as it is out of scope for this PR